File: Backend-POS/POS.Main/POS.Main.Business.Order/Models/OrderBill/OrderBillResponseModel.cs
namespace POS.Main.Business.Order.Models.OrderBill;

public class OrderBillResponseModel
{
    public int OrderBillId { get; set; }
    public int OrderId { get; set; }
    public string BillNumber { get; set; } = string.Empty;
    public string BillType { get; set; } = string.Empty;
    public decimal SubTotal { get; set; }
    public decimal TotalDiscountAmount { get; set; }
    public decimal NetAmount { get; set; }
    public decimal ServiceChargeRate { get; set; }
    public decimal ServiceChargeAmount { get; set; }
    public decimal VatRate { get; set; }
    public decimal VatAmount { get; set; }
    public decimal GrandTotal { get; set; }
    public string Status { get; set; } = string.Empty;
    public DateTime? PaidAt { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: Frontend-POS/RBMS-POS-Client/src/app/core/api/models/order-bill-response-model.ts
/* tslint:disable */
/* eslint-disable */
export interface OrderBillResponseModel {
  billNumber?: string | null;
  billType?: string | null;
  createdAt?: string;
  grandTotal?: number;
  netAmount?: number;
  orderBillId?: number;
  orderId?: number;
  paidAt?: string | null;
  serviceChargeAmount?: number;
  serviceChargeRate?: number;
  status?: string | null;
  subTotal?: number;
  totalDiscountAmount?: number;
  vatAmount?: number;
  vatRate?: number;
}
